snykTask/src/task-args.ts

```diff
@@ -1,5 +1,9 @@
 import * as tl from 'azure-pipelines-task-lib';
-
+import {
+  isNotValidThreshold,
```

Comment:
Since you moved validation logic here, `isNotValidThreshold` should be also a part of TaskArgs class now.